### PR TITLE
build(docker-client): Add support for building the cpp-client

### PR DESCRIPTION
**Pull request by oleorhagen**
Ticket: MEN-5796

Signed-off-by: Ole Petter <ole.orhagen@northern.tech>

### CODE REVIEW

**Comment by oleorhagen** [comment]
Follow up on: https://github.com/mendersoftware/integration/pull/2300

**Comment by kacf** [comment]
I suggest @lluiscampos reviews this one since he already looked into this on master.

File: extra/mender-client-docker-addons/Dockerfile [hunk 1]
```diff
@@ -38,12 +42,12 @@ RUN mender-artifact write bootstrap-artifact \
         --output-path /bootstrap.mender
 
 WORKDIR /src/mender-connect
-RUN make prefix=/mender-install install
+RUN DESTDIR=/mender-install make prefix=/usr install
```
Comment (oleorhagen) [inline extra/mender-client-docker-addons/Dockerfile:45]: I'm guessing you want this reverted to the old setup here?

But in a sense, it bothers me that we don't do this properly.

Would it be off if I fixed this in the given repos?

**Review by lluiscampos — CHANGES_REQUESTED** [review]

**Comment by oleorhagen** [comment]
Obviously the `extra-tools` test failes. This is pre-existing. I can fix.

Question is rather. Do we want to?

**Comment by lluiscampos** [comment]
> Obviously the `extra-tools` test failes. This is pre-existing. I can fix.
> 
> Question is rather. Do we want to?

@oleorhagen Maybe just disable this CI job for the feature branch.

With regards to the license update,

What you should do is actually merge the current master into the feature branch (or rather fast-forward if we just created it now after this issue) so that you get the license update and also the apply + revert of this `Dockerfile`, so that you new commit goes on top of that and we don't get the conflict later.

**Review by lluiscampos — APPROVED** [review]

**Review by lluiscampos — APPROVED** [review]

**Pull request merged**